crawl-ref/source/dat/des/branches/zot.des

```diff
@@ -2786,6 +2786,41 @@ MAP
 ............?............
 ENDMAP
 
+NAME:   dreamdust_zin_vs_xom
+TAGS:   no_monster_gen
+ORIENT: float
+DEPTH:  Zot, !Zot:$
+COLOUR: v = white
+TILE:   v = dngn_metal_wall_white
+FTILE:  _`1 = floor_w_marble
+KFEAT:  _ = altar_zin
+TILE:   G = dngn_statue_angel
+MONS:   angel god:zin dbname:zin_angel, \
+        killer klown-shaped pillar of salt, \
```

Comment:
Recommend defining "terrain" monsters like this with KMONS on an appropriate glyph like `a` or `p` so it doesn't get confused with the list of monsters the player actually fights on `1-7`.